app/assets/stylesheets/content/_flash_messages.sass

```diff
@@ -42,6 +42,7 @@
 
 .flash
   display: none
+  padding: 15px 5px
```

Comment:
Just an opinion but I believe that at least the vertical padding should be specified with `rem` instead of `px` so that when the font size is changed, the padding adjusts itself nicely.